addons/web/tests/test_assets.py

```diff
@@ -15,14 +15,12 @@
 class TestAssetsGenerateTimeCommon(odoo.tests.TransactionCase):
 
     def generate_bundles(self):
-        bundles = set()
         installed_module_names = self.env['ir.module.module'].search([('state', '=', 'installed')]).mapped('name')
-        for addon_path in odoo.addons.__path__:
-            for addon in installed_module_names:
-                manifest = read_manifest(addon_path, addon) or {}
-                assets = manifest.get('assets')
-                if assets:
-                    bundles |= set(assets.keys())
+        bundles = {
+            key
+            for module in installed_module_names
+            for key in load_manifest(module).get('assets') or {}
```

Comment:
Hey, `load_manifest` starts from a copy of a default manifest, which has `assets`
```suggestion
            for key in load_manifest(module)['assets']
```